docker/grafana/datasource.local.yml

```diff
@@ -4,6 +4,6 @@ datasources:
   - name: Prometheus
     type: prometheus
     access: proxy
-    url: http://localhost:9090
+    url: http://host.docker.internal:9090
```

Comment:
Why is localhost:9090 not reachable in MacOS?

Reply:
I think that has to do with the fact that docker on MacOS does not have access to the host directly and it sits on top of a linux vm. Which makes it impossible to use `localhost` to refer to the actual host.

Reply:
Then I would move this changes to another PR to keep this one in scope to `dadepo/retry-el-executepayload`

Reply:
This changes was made due to the request to add metrics to the retry mechanism. Since it makes it easier to see the metrics via the Prometheus/Grafana setup locally, instead of having to first push and run on a linux server. 

But I can move it to a separate PR as you suggested and just use the /metrics endpoint exposed by the node